docs/docs/api.md

```diff
@@ -35,13 +36,21 @@ response was a 4xx or 5xx.)
 
 ### `GET /v0/markets`
 
-Lists all markets.
+Lists all markets, ordered by creation date descending.
+
+Parameters:
+
+- `n`: Optional. How many markets to return. The maximum and the default is 1000.
```

Comment:
Slight preference for `limit` over `n`; a bit more clear. It's what Stripe uses: https://stripe.com/docs/api/pagination

Reply:
Also not sure that `before` is optimal; Stripe has `ending_before`? But it's kind of weird since our result is creation date with newest first.

Reply:
I'm definitely not going to argue that these are optimal, but few things about our API are optimal. We will just have to expect to do a big pass at some point in the future with breaking changes to make the "v1" API.

Reply:
Yeah I think "before" is fine, and I'd just do "limit" for clarity